pipenv/core.py

```diff
@@ -1889,12 +1889,16 @@ def do_install(
     # Capture -e argument and assign it to following package_name.
     more_packages = list(more_packages)
     if package_name == '-e':
+        if not more_packages:
+            raise click.BadArgumentUsage('Please provide path to setup.py')
```

Comment:
This message could be misleading. We don’t want a path to setup.py, but to *the directory containing it*. I would use something like “path to editable package (e.g. the directory containing setup.py)” instead.